lib/rspecq/worker.rb

```diff
@@ -25,6 +31,7 @@ def initialize(build_id:, worker_id:, redis_host:, files_or_dirs_to_run:)
       @populate_timings = false
       @file_split_threshold = 999999
       @heartbeat_updated_at = nil
+      @max_requeues = DEFAULT_MAX_REQUEUES
```

Comment:
Ditto:

```ruby
@max_requeues = 3
```

In the future, it may make sense to have all defaults grouped somewhere in some kind of constant.